meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,144 @@
+   - Patch release for libp2p-switch (0.40.8) for dependency updates, including multistream-select already piped fix version bump.   
+ - Blocked:
+   - Delegated routing support in [go-ipfs#4594](https://github.com/ipfs/go-ipfs/pull/4595).
+ - Next:
+   - libp2p-tcp transport isnt currently dialble over dns, looking into this (mafmt needs to be updated for this to work)
+   - Continue with state machine updates for libp2p connections
+
+@vasco-santos
+- Done:
+  - IPNS over Pubsub OKR
+    - Created `js-datastore-pubsub` module
+      - Initial implementation of [ipfs/js-datastore-pubsub#1](https://github.com/ipfs/js-datastore-pubsub/pull/1)
+    - `js-ipns`
+      - Added records validator [js-ipns#5](https://github.com/ipfs/js-ipns/pull/5)
+      - Added routing key format [js-ipns#7](https://github.com/ipfs/js-ipns/pull/7)
+      - PRs approved, merged and released
+    - `js-ipfs`
+      - Implementation almost done 
+  - fix: libp2p records for IPNS should be signed
+    - https://github.com/ipfs/js-ipfs/pull/1543
+- Blocked:
+  - IPNS
+    - [interop#26](https://github.com/ipfs/interop/pull/26)
+      - Needs js-ipfs new release with IPNS, and to be reviewed
+  - IPNS over Pubsub
+    - [interface-ipfs-core#361](https://github.com/ipfs/interface-ipfs-core/pull/361)
+        - Needs to be approved, merged and released
+      - [js-ipfs-api](https://github.com/ipfs/js-ipfs-api/pull/846)
+        - Needs the previous PR released, followed by being approved, merged and released
+- Next:
+  - Fix code review for `js-datastore-pubsub`
+  - Add tests for `js-ipfs` implementation of `IPNS over Pubsub` and create PR
+  - Fix IPNS over Pubsub PRs feedback
+  - Create interop tests for `IPNS over Pubsub`
+  - Create tests for testing DHT interop  
+
+@hugomrdias
+- Done
+  - ipfs chunked add
+    - [#1540](https://github.com/ipfs/js-ipfs/pull/1540)
+  - ipfs-api chunked add
+    - [#851](https://github.com/ipfs/js-ipfs-api/pull/851)
+    - [stream-http/tree/fix/body-handling](https://github.com/hugomrdias/stream-http/tree/fix/body-handling)
+- Blocked
+- Next
+  - finish add chunked
+  - fix `stream-http`
+    - Doesn't work inside a service-worker on Firefox [848#issuecomment-419887279](https://github.com/ipfs/js-ipfs-api/issues/848#issuecomment-419887279)
+  - add api add chunked to big data tests
+  - finish chore/improve-docs branch documentation.js for some reason doesn't find `factory-daemon`. `npx documentation serve ./js-ipfsd-ctl/src/index.js -w -f html`
+  - PR aegir to update documentation.js
+
+
+@travisperson
+ - Done:
+   - iptb [go/js/browser demo](https://asciinema.org/a/zn07AUFgZ5RWzAwit7mjpdqvW) 
+ - Blocked:
+ - Next:
+   - Merge iptb browser plugin
+   - Build test harness
+
+@vmx
+ - Done:
+   - Conference (FOSS4G) and travels in Tanzania
```

Comment:
Is there a url to the talk??

Reply:
It's not easy to link to it, here's the best one I could find: https://fkih7q.attendify.io/schedule.html#9YYK6haD49GaQsORLc